### PR TITLE
Add config.xml to downloaded TAU project

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] http://suprem.sec.samsung.net/jira/browse/TIZENWF-2567
[Problem] Deasign Editor does not start and tizen packaging does not
               generate wgt file.
[Solution] Add config.xml and fill its content
[Remarks] Although config.xml is added, DE still does not start.
                This issue is reported http://suprem.sec.samsung.net/jira/browse/TIZENWF-2580
[TEST]
    1. Create sample html with the following content:
      <html>
        <body>
          <script>
            window.open("http://localhost:3000/demos/?path=" + encodeURIComponent("mobile/UIComponents/components/controls/checkbox.html"));
          </script>
        </body>
      </html>
      Alternatively, you can manually encode path by replacing '/' with '%2F'.
    
    2. config.xml should exist in root directory.

### CODE REVIEW

File: libs/util.js [hunk 1]
```diff
@@ -57,4 +60,46 @@ function authenticateUser(req, res, next, user) {
     }
     next();
   });
-}
+};
+
+exports.createConfigXML = function(dstPath, configPreferences, configContent, callback) {
+  console.log(JSON.stringify(configContent));
```
Comment (lslachciak) [inline libs/util.js]: is this log needed?

File: routes/demos.js [hunk 2]
```diff
@@ -5,26 +5,25 @@ const exec = require('child_process').exec;
...
     (req, res) => {
       let projectId;
       async.waterfall([
 
         // Create new project.
         callback => {
           const newProject = new Project();
-          newProject.name = "Tau Sample Demo";
-          newProject.type = "web";
+          newProject.name = 'TauSampleDemo';
```
Comment (lslachciak) [inline routes/demos.js]: why spaces removed?

File: libs/util.js (hunk 1 above)
Comment (grzegorz-czajkowski) [reply libs/util.js]: It is not needed. Removed.

File: routes/demos.js (hunk 2 above)
Comment (grzegorz-czajkowski) [reply routes/demos.js]: Restored spaces.

**Comment by lslachciak** [comment]
works well tested with uris like:
http://localhost:3000/demos/?path=mobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fcheckbox.html

http://localhost:3000/demos/?path=mobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fradio.html

**Pull request merged**